### PR TITLE
Copy updates and a small CSS tweak

### PR DESCRIPTION
**Pull request by goldenmeanie**

### CODE REVIEW

File: src/sass/edu-benefits.scss [hunk 1]
```diff
@@ -159,6 +168,7 @@ ul.claim-list{
 // Testing a slightly modified required asterisk
 .form-required-span {
   font-size: 1.7em;
+  list-style: 1;
```
Comment (jbalboni) [inline src/sass/edu-benefits.scss]: Is this intentional?

**Review by jbalboni — APPROVED** [review]
LGTM, one question

File: src/sass/edu-benefits.scss (hunk 1 above)
Comment (goldenmeanie) [inline src/sass/edu-benefits.scss]: Yes, it fixes the extra space between that was getting added between inputs.

Comment (goldenmeanie) [inline src/sass/edu-benefits.scss]: Oh. Nope, bad auto-complete! Should have been line-height: 1;

**Pull request merged**